src/modules/display.js

```diff
@@ -0,0 +1,31 @@
+import likeSend from './sendlikes.js';
+import getLike from './getLikes.js';
+import itemCounter from '../test/itemcounter.js';
+
+const display = (array) => {
+  const main = document.querySelector('.main');
+  main.innerHTML = '';
+  array.forEach((item) => {
+    main.innerHTML += `<div class="box">
+    <img class='image' src="${item.keyImages[0].url}" alt="">
+    <h3 class="title">${item.title}</h3>
+    <div class="comment-like">
+    <div>
+      <button class="comment-button ${item.id}" id ="com-id" >Comments</button>
+    </div>
+      <div class="like-div">
+      <button class="like-button" id='${item.id}'>
+      <i class="fa fa-heart"></i>
+      </button>
```

Comment:
### [OPTIONAL]
When I click on the 'likes' (text) it updates the likes counter and when I click on the icon it doesn't update the likes counter. That's why the previous Code Reviewer complained, the like button sometimes works and sometimes doesn’t. 

Overall, the functionality is working. 👍